src/components/SuggestionsList/__snapshots__/SuggestionsListItem.test.tsx.snap

```diff
@@ -43,8 +41,6 @@ exports[`<SuggestionsListItem /> when isSelected=true renders as expected 1`] =
   <MediaObject
     imageContent={
       <Avatar
-        borderType="round"
-        imageShouldFadeIn={false}
```

Comment:
I don't fully understand how having these in Avatar's default props resulted them being in the snapshot here.

Reply:
that's just how snapshots work for all React components, it'll always show default props